tests/test_consumer.py

```diff
@@ -72,6 +74,7 @@ def write_step() -> ProcessedMessageBatchWriter:
             ),
         )
 
+    health_check_file = tempfile.mktemp()
```

Comment:
you can use the pytest tmpdir fixture here

Reply:
fixed